packages/website/components/PageContent.tsx

```diff
@@ -0,0 +1,162 @@
+import React from 'react';
+import tokens from '@contentful/f36-tokens';
+import { css } from 'emotion';
+import {
+  DisplayText,
+  Badge,
+  TextLink,
+  Stack,
+  Heading,
+} from '@contentful/f36-components';
+
+import storybookIcon from '../resources/icons/storybook.svg';
+import githubIcon from '../resources/icons/github.svg';
+
+const styles = {
+  root: css`
+    display: 'flex';
+    flex-direction: 'column';
+    width: 960px;
+    margin: 0 auto;
+    padding-top: ${tokens.spacing2Xl};
+  `,
+  header: css`
+    width: 960px;
+    margin: 0 auto;
+    padding-bottom: ${tokens.spacingXl};
+    margin-bottom: ${tokens.spacingXl};
+    border-bottom: 1px solid ${tokens.gray300};
```

Comment:
the documentation pages with `v4Doc: true` should not have this border

In those pages we should only show the DisplayText

Reply:
shouldn't we delete `v4Doc` attributes now? Why not to have border-bottom after all titles?

Reply:
Another way to make this question is: why having them? hahahah

Anyway, it was just how @domarku designed them
if he is happy re-introducing the border-bottom, I will be fine as well

Reply:
and the idea of the `v4Doc` attribute is because we are in this transition period where some docs follow the new format and some docs don't

Reply:
I think we can take each page separately lately, we have tasks for them, right?

Reply:
We have tasks to adapt each page to the new format yes